boards/sparkfun/artemis_nano/Kconfig.artemis_nano

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2023 Ambiq Micro Inc. <www.ambiq.com>
```

Comment:
I think the copyright should be corrected to yours.

Reply:
Copyright has been updated